lib/v2/screens/settings/edit_name/edit_name_screen.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/v2/components/flat_button_long.dart';
+import 'package:seeds/v2/screens/settings/edit_name/viewmodels/bloc.dart';
+import 'package:seeds/i18n/edit_name.i18n.dart';
+
+class EditNameScreen extends StatefulWidget {
+  const EditNameScreen({Key key}) : super(key: key);
+
+  @override
+  _EditNameScreenState createState() => _EditNameScreenState();
+}
+
+class _EditNameScreenState extends State<EditNameScreen> {
+  EditNameBloc _editNameBloc;
+  final _nameController = TextEditingController();
+  final _formKeyPassword = GlobalKey<FormState>();
+
+  @override
+  void initState() {
+    super.initState();
+    _editNameBloc = EditNameBloc();
+    _nameController.text = 'Raul';
+    _nameController.addListener(_onNameChanged);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: AppColors.primary,
```

Comment:
scaffold color will no longer be needed as soon as we merge gabriels PR, please remove this. and any other scaffold background color that has been added